### ISSUE TITLE
Ruby 3.0 RBS support

### ISSUE THREAD

**Issue by longhoangwkm** [vim-ruby/vim-ruby#426]
Do we have plan to support format .rbs file

**Comment by dkearns** [comment]
Yes, but we might not get to it for a while.   We have until Santa flies again...

I have made a start on the syntax file.

**Comment by calebhearth** [comment]
@jlcrochet put out a plugin that adds syntax and indentation for RBS files: https://github.com/jlcrochet/vim-rbs
